react/components/InfoCard/LinkWrapper.js

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from 'react'
+import { Link } from 'vtex.render-runtime'
+
+const LinkWrapper = ({ imageActionUrl, children, extraCondition, linkProps }) => {
```

Comment:
A ternary there would look better in my opinion. Just personal taste, change if you feel like it 😉

Reply:
I will keep early return since IMO it makes the code more clear and clean in this situation.